sdk/cosmos/pom.xml

```diff
@@ -14,6 +14,9 @@
       <id>coverage</id>
       <modules>
         <module>azure-cosmos</module>
+        <module>azure-spring-data-cosmos</module>
+        <module>azure-cosmos-spark_3_2-12</module>
+        <module>azure-cosmos-encryption</module>
```

Comment:
The modules will also need to be added as dependencies in the `dependencies` configuration below in the same way `azure-cosmos` was added. Jacoco needs to have the SDK as a dependency to generate aggregate coverage correctly.